components/IdentifyViewer.jsx

```diff
@@ -525,6 +560,21 @@ class IdentifyViewer extends React.Component {
         } else {
             body = (
                 <div className="identify-flat-results-list">
+                    <div className="identify-selectbox">
+                        <select className="identify-layer-select"
+                            onChange={(e) => {const selectedLayer = e.target.value; this.setState({ selectedLayer });}}
+                        >
+                            <option value="all">{LocaleUtils.tr("identify.layerall")}</option>
```

Comment:
There is a theoretical possibility of a name collision with a layer named `all`. I'd just use an empty string.

Reply:
Thansk, I've fixed that